pandas/tests/test_window.py

```diff
@@ -3019,73 +3019,6 @@ def test_rolling_max_gh6297(self):
         x = series.resample('D').max().rolling(window=1).max()
         tm.assert_series_equal(expected, x)
 
-    def test_rolling_max_how_resample(self):
-
-        indices = [datetime(1975, 1, i) for i in range(1, 6)]
-        # So that we can have 3 datapoints on last day (4, 10, and 20)
-        indices.append(datetime(1975, 1, 5, 1))
```

Comment:
just remove the actual how= params, the rest of the tests are ok (you can rename this slightly as well). its possible these tests are somewhat duplicated as well. have a look.

Reply:
I think these test look a bit contrieved then ATM, but ok, I will look into them when I refactor ``pd.stats`` to see if it makes sense to keep them, possibly in an altered shape.